lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/dbo/schema/JsonSchemaDaoImpl.java

```
@@ -297,6 +300,21 @@ public JsonSchemaVersionInfo createNewSchemaVersion(NewSchemaVersionRequest requ
 		return info;
 	}
 
+	@Override
+	public Iterator<Long> getObjectIdsBoundToSchemaIterator(String schemaId) {
+		return new PaginationIterator<Long>((long limit, long offset) -> {
+			return getNextPageForEntitiesBoundToSchema(schemaId, limit, offset);
+		}, PAGE_SIZE_LIMIT);
+	}
+	
+	private List<Long> getNextPageForEntitiesBoundToSchema(String schemaId, long limit, long offset) {
```

test this to ensure limit and offset are wired as expected.

---

ValidateArgs.() schemaId should throw an exception